File: Agro-Ecommerce-Backend/src/main/java/com/agro/model/Return.java
package com.agro.model;

import java.time.LocalDateTime;

import com.fasterxml.jackson.annotation.JsonIgnore;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name="returns")
public class Return {

	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

//	@JsonIgnore
    @ManyToOne
    @JoinColumn(name = "order_id", nullable = false)
    private Order order;


//	@JsonIgnore
    @ManyToOne
    @JoinColumn(name = "user_id", nullable = false)
    private User user;

    private String reason;
    private String status;
    private LocalDateTime requestDate;
}

File: agro_ecommerce_flutter/lib/app/models/Return.dart
class Return{

  final int id;
  final int orderId;
  final int userId;
  final String reason;
  final String status;
  final DateTime requestDate;

  Return({
    required this.id,
    required this.orderId,
    required this.userId,
    required this.reason,
    required this.status,
    required this.requestDate
  });

  factory Return.fromJson(Map<String, dynamic> json) {
    return Return(
      id: json['id'],
      orderId: json['orderId'],
      userId: json['userId'],
      reason: json['reason'],
      status: json['status'],
      requestDate: json['requestDate'] is String ? DateTime.parse(json['requestDate']) : json['requestDate']
    );
  }
}